docker/pravega/Dockerfile

```diff
@@ -16,7 +16,16 @@ RUN apt-get update && apt-get install -y -q \
     jq \
     curl \
     net-tools \
-    iproute2  \
+    iproute2
+
+# Adding Java system truststore 'cacerts', as it is missing from the Java
+# distribution installed by the base image and one of the storage bindings(ECS)
+# depends on its presence.
+#
+# Without `mkdir -p /usr/share/man/man1` we hit the issue mentioned in:
+# https://stackoverflow.com/questions/58160597/docker-fails-with-sub-process-usr-bin-dpkg-returned-an-error-code-1
```

Comment:
I'm not sure that referring to SO is the best approach here. Could we just summarize the reason in 1 or 2 lines?

Reply:
Removed the SO link and added comments.